cmd/kata-pkgsync/config.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 SUSE LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+obsprojects:
+    home:katacontainers:releases:
+
+        releases:
+            - master
+            - stable-1.8
```

Comment:
I'm guessing we're going to need some documentation regarding maintaining this list.